Forwarded: https://github.com/mariadb-corporation/libmarias3/pull/111
Origin: https://patch-diff.githubusercontent.com/raw/mariadb-corporation/libmarias3/pull/111.patch
Author: Otto Kekäläinen <otto@kekalainen.net>
Date: Sat, 11 Mar 2023 12:05:28 -0800
Subject: [PATCH] Fix spelling: occured -> occurred

All new code of the whole pull request, including one or several files
that are either new files or modified ones, are contributed under the
BSD-new license. I am contributing on behalf of my employer Amazon Web
Services, Inc.
---
 docs/api/functions.rst | 10 +++++-----
 src/xml.c              |  2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

--- a/storage/maria/libmarias3/docs/api/functions.rst
+++ b/storage/maria/libmarias3/docs/api/functions.rst
@@ -147,7 +147,7 @@ Example
    res= ms3_list(ms3, s3bucket, NULL, &list);
    if (res)
    {
-       printf("Error occured: %d\n", res);
+       printf("Error occurred: %d\n", res);
        return;
    }
    list_it= list;
@@ -220,7 +220,7 @@ Example
    res= ms3_put(ms3, s3bucket, "test/ms3.txt", (const uint8_t*)test_string, strlen(test_string));
    if (res)
    {
-       printf("Error occured: %d\n", res);
+       printf("Error occurred: %d\n", res);
        return;
    }
    ms3_deinit(ms3);
@@ -290,7 +290,7 @@ Example
    res= ms3_get(ms3, s3bucket, "test/ms3.txt", &data, &length);
    if (res)
    {
-       printf("Error occured: %d\n", res);
+       printf("Error occurred: %d\n", res);
        return;
    }
    printf("File contents: %s\n", data);
@@ -348,7 +348,7 @@ Example
    res = ms3_delete(ms3, s3bucket, "test/ms3.txt");
    if (res)
    {
-       printf("Error occured: %d\n", res);
+       printf("Error occurred: %d\n", res);
        return;
    }
    ms3_deinit(ms3);
@@ -384,7 +384,7 @@ Example
    res= ms3_status(ms3, s3bucket, "test/ms3.txt", &status);
    if (res)
    {
-       printf("Error occured: %d\n", res);
+       printf("Error occurred: %d\n", res);
        return;
    }
    printf("File length: %ld\n", status.length);
--- a/storage/maria/libmarias3/src/xml.c
+++ b/storage/maria/libmarias3/src/xml.c
@@ -800,7 +800,7 @@ node_creation:;
 	return node;
 
 
-	/* A failure occured, so free all allocalted resources
+	/* A failure occurred, so free all allocalted resources
 	 */
 exit_failure:
 	if (tag_open) {
